scripts/travis_prepare.sh

```diff
@@ -147,8 +148,3 @@ if (( "$TEST_PAST" )); then
     create 9.2
     create 9.3
 fi
-
-# Postgres built from master
-if (( "$TEST_FUTURE" )); then
```

Comment:
Can you please leave this stanza here to remember how to add back 14? :)

Reply:
Sure! :)

Reply:
Done!